File: src/BukkitPE/level/generator/normal/biome/SnowyBiome.php
<?php
namespace BukkitPE\level\generator\normal\biome;

use BukkitPE\block\Block;

abstract class SnowyBiome extends NormalBiome{

	public function __construct(){
		$this->setGroundCover([
			Block::get(Block::SNOW_LAYER, 0),
			Block::get(Block::GRASS, 0),
			Block::get(Block::DIRT, 0),
			Block::get(Block::DIRT, 0),
			Block::get(Block::DIRT, 0),
		]);
	}
}

File: src/main/java/net/BukkitPE/level/generator/biome/SnowyBiome.java
package net.BukkitPE.level.generator.biome;

import net.BukkitPE.block.Block;
import net.BukkitPE.block.BlockDirt;
import net.BukkitPE.block.BlockGrass;
import net.BukkitPE.block.BlockSnowLayer;

/**

 * BukkitPE Project
 */
public abstract class SnowyBiome extends NormalBiome {
    public SnowyBiome() {
        this.setGroundCover(new Block[]{
                new BlockSnowLayer(),
                new BlockGrass(),
                new BlockDirt(),
                new BlockDirt(),
                new BlockDirt()
        });
    }
}
